src/backend/mpi/swdsm.cpp

```diff
@@ -1073,19 +1079,19 @@ void argo_initialize(unsigned long long size){
 	MPI_Comm_create(MPI_COMM_WORLD,workgroup,&workcomm);
 	MPI_Group_rank(workgroup,&workrank);
 
-	if(size < pagesize*numtasks){
-		size = pagesize*numtasks;
+	if(argo_size < pagesize*numtasks){
+		argo_size = pagesize*numtasks;
 	}
```

Comment:
For future patch: This could be replaced with `std::max`.

Reply:
See issue #24